analyses/selection-strategy-comparison/01-selection-strategies.rmd

```diff
@@ -5,6 +5,9 @@ output:
   html_notebook:
     toc: yes
     toc_float: yes
+date: Sys.Date()
+params:
+  neighbors: 15
```

Comment:
I'm assuming this is overridden by the command line option. I'm pretty sure this is what's happening but I'm voicing my thoughts so you can correct me if I'm wrong.